site/docs/reference/Connectors/capture-connectors/PostgreSQL.md

```diff
@@ -17,6 +17,14 @@ To use this connector, you'll need a PostgreSQL database setup with the followin
 
 ### Setup
 
+:::info
+These setup instructions are PostgreSQL instances you manage yourself. If you use a cloud-based managed service
+for your database, different setup steps are required.
```

Comment:
Perhaps replace "are required" with "may be required"? There are probably some managed PostgreSQL offerings where the normal setup steps would still work, it's just that Amazon RDS specifically has tweaked a couple of important details with their own changes, and if I read "may be" then I'm more inclined to go try it anyway.